### DISCUSSION TITLE
Flathub Quality Guidelines

### DISCUSSION THREAD

**Discussion by cassidyjames** [zen-browser/desktop#2646] [General]
I'm a volunteer with [Flathub](https://flathub.org/), and one thing I do is help review app metadata to check if they meet the [Flathub Quality Guidelines](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines). These guidelines are technically optional for apps _to be on Flathub_, but are required for apps to get featured on the home page (e.g. in the carousel and app of the day banners).

I'd love to help get [Zen's Flathub listing](https://flathub.org/apps/io.github.zen_browser.zen) into shape so we can feature it!

I only see the following issues:

- **Screenshots:** Screenshots should be of just the window (e.g. no wallpaper behind) and at a decent size so everything is easily viewable on at the sizes presented at flathub.org. Some examples:

   ![Screenshot from 2024-11-04 14-42-24](https://github.com/user-attachments/assets/34420fc6-e359-4a25-8b5e-13f40f697e52)

   ![Screenshot from 2024-11-04 15-27-07](https://github.com/user-attachments/assets/7bf82612-718b-423a-a1ab-cb48cde7dc1a)

- **Brand colors**: The current brand colors are `#d9d9d9` for light style and `#f5f5f5` for the dark style; I'd recommend a much darker color for the dark style, as a white banner is going to be really harsh. You can use the [Flathub banner preview page](https://docs.flathub.org/banner-preview/) to play with colors, but I think something around `#eeeeee` and `#444444` could look nice for the light and dark styles respectively:

   ![image](https://github.com/user-attachments/assets/a0d3382b-dc16-4195-9629-6b5ef00c3d67)

   Alternatively, I saw some hints of pink/purple on your site, so something in that vein could be really eye-catching (…but might not be what you're going for):

   ![image](https://github.com/user-attachments/assets/419b6527-8e84-4a46-bbfc-dff91c35c993)

Other than that, it looks like Zen is passing the metadata quality checks—nice work! I would recommend including the word "browser" in your listing explicitly as well to make it easier to search for, but that's not strictly required at all. :)

I'd love to get Zen featured on the home page of Flathub soon if we can have you address those items; let me know if I can help clarify anything, as well!

**Comment by TylerLaBree** [comment]
I took some screenshots which should work well on Flathub

![main-light](https://github.com/user-attachments/assets/25506916-75ce-41f3-9547-8d1acf36ddac)

![main-dark](https://github.com/user-attachments/assets/151620ea-5e2d-4f57-abf7-cc89ee1646c0)

![glance-light](https://github.com/user-attachments/assets/f2ff6d29-11c1-4a08-9671-c24272bbf388)

![glance-dark](https://github.com/user-attachments/assets/2dda6895-4b8a-4b9b-8033-4a168242f5b5)

![split-light](https://github.com/user-attachments/assets/08fc16e9-d735-4428-a503-2a858277eb6b)

![split-dark](https://github.com/user-attachments/assets/d1dd0b21-75d0-49af-8cb9-7ce053c90e11)

**Comment by cassidyjames** [comment]
Here's a suggestion for the brand colors that I think would pass the guidelines:

```xml
<branding>
  <color type="primary" scheme_preference="light">#f2f0e3</color>
  <color type="primary" scheme_preference="dark">#2e2e2e</color>
</branding>
```

![Screenshot 2025-02-25 at 16-28-20 Brand color preview Flathub Documentation](https://github.com/user-attachments/assets/87723e27-ca93-47e4-ad56-6b4a2f9a0f11)
